tests/integration/queries/test_migration_progress.py

```diff
@@ -188,6 +191,110 @@ def policies() -> list[PolicyInfo]:
+                LineageAtom(object_type="QUERY", object_id=f"my_dashboard_id/{query.id}", other={"name": "my_query"}),
+            ],
+            assessment_start_timestamp=dt.datetime.now(dt.timezone.utc) - dt.timedelta(minutes=5.0),
+            assessment_end_timestamp=dt.datetime.now(dt.timezone.utc) - dt.timedelta(minutes=2.0),
+        ),
+    ]
+    return records
+
+
+@pytest.fixture
+def used_tables(make_workspace_file, make_table) -> list[UsedTable]:
+    table = make_table(catalog_name="hive_metastore")
+    workspace_file = make_workspace_file(content=f'df = spark.read.table("{table.full_name}")\ndisplay(df)')
+    records = [
+        UsedTable(
+            catalog_name=table.catalog_name,  # This table is pending migration
+            schema_name=table.schema_name,
+            table_name=table.name,
+            is_read=False,
+            # Technically, the mocked code is reading the table, but marking it as write allows us to set the owner to
+            # the current user, which we can test below.
+            is_write=True,
+            source_id=str(workspace_file),
+            source_timestamp=dt.datetime.now(dt.timezone.utc) - dt.timedelta(hours=2.0),
+            source_lineage=[
+                LineageAtom(object_type="WORKFLOW", object_id="my_workflow_id", other={"name": "my_workflow"}),
+                LineageAtom(object_type="TASK", object_id="my_workflow_id/my_task_id"),
+                LineageAtom(object_type="NOTEBOOK", object_id="my_notebook_path"),
+                LineageAtom(object_type="FILE", object_id=str(workspace_file)),
+            ],
+            assessment_start_timestamp=dt.datetime.now(dt.timezone.utc) - dt.timedelta(minutes=5.0),
+            assessment_end_timestamp=dt.datetime.now(dt.timezone.utc) - dt.timedelta(minutes=2.0),
+        ),
+        UsedTable(
+            catalog_name="catalog",  # This table is migrated
+            schema_name="staff_db",
+            table_name="employees",
+            is_read=False,
+            is_write=True,
+            source_id=str(make_workspace_file()),
+            source_timestamp=dt.datetime.now(dt.timezone.utc) - dt.timedelta(hours=2.0),
+            source_lineage=[
+                LineageAtom(object_type="WORKFLOW", object_id="my_workflow_id", other={"name": "my_workflow"}),
+                LineageAtom(object_type="TASK", object_id="my_workflow_id/my_task_id"),
+                LineageAtom(object_type="NOTEBOOK", object_id="my_notebook_path"),
+                LineageAtom(object_type="FILE", object_id="my file_path"),
+            ],
+            assessment_start_timestamp=dt.datetime.now(dt.timezone.utc) - dt.timedelta(minutes=5.0),
+            assessment_end_timestamp=dt.datetime.now(dt.timezone.utc) - dt.timedelta(minutes=2.0),
+        ),
+    ]
+    return records
+
+
 @pytest.fixture
 def catalog_populated(  # pylint: disable=too-many-arguments
```

Comment:
Expand name to be more explicit about that it is populated with the data fixture above, e.g. rename to `make_catalog_with_test_data(catalog_name: str)`

Reply:
If a fixture is longer than 10 lines, it should be `make_`